crates/web5_cli/src/main.rs

```diff
@@ -13,6 +15,9 @@ use clap::{Parser, Subcommand};
 struct Cli {
     #[command(subcommand)]
     command: Commands,
+    /// A file to output command output to.
+    #[arg(long, global = true)]
```

Comment:
Because you're passing the `sink` through to each subcommand, do we need `global = true`?

Reply:
Makes sense!